107/src/test/resources/ehcache-107-default-copiers.xml

```diff
@@ -35,4 +40,29 @@
     <ehcache:heap>10</ehcache:heap>
   </ehcache:cache-template>
 
+  <ehcache:cache-template name="stringtemplate">
```

Comment:
Well that XML is about testing the configuration of a default copier while your test is about getting smarter defaults when none configured.Not the same to me.